python/src/trezorlib/firmware/core.py

```diff
@@ -126,7 +126,10 @@ class FirmwareImage(Struct):
 
     @staticmethod
     def calc_padding(hw_model: bytes, len: int) -> int:
-        alignment = Model.from_hw_model(hw_model).code_alignment()
+        try:
+            alignment = Model.from_hw_model(hw_model).code_alignment()
+        except ValueError:
+            alignment = Model.T3W1.code_alignment()
```

Comment:
Would it be clearer to add an explicit `DEFAULT_CODE_ALIGNMENT` constant (instead of re-using T3W1 alignment)?

Reply:
based on this comment, the whole function was removed :)